sprint-mission-7/queries.sql

```diff
@@ -0,0 +1,162 @@
+/*
+  다음 경우들에 대해 총 14개의 SQL 쿼리를 작성해 주세요.
+  예시로 값이 필요한 경우 적당한 값으로 채워넣어서 작성하면 됩니다. 
+*/
+
+/*
+  1. 내 정보 업데이트 하기
+  - 닉네임을 "test"로 업데이트
+  - 현재 로그인한 유저 id가 1이라고 가정
+*/
+UPDATE users
+SET nickname = 'test',
+    updated_at = now()
+WHERE user_id = 1;
```

Comment:
선언한 필드명과 다릅니다. 선언한 필드를 사용하도록 해주세요.

```suggestion
WHERE id = 1;
```